Sources/DDGSync/internal/SyncRequestMaker.swift

```diff
@@ -54,8 +55,10 @@ struct SyncRequestMaker: SyncRequestMaking {
             throw SyncError.unableToEncodeRequestBody("Sync PATCH payload is not a valid JSON")
         }
 
-        let body = try JSONSerialization.data(withJSONObject: json, options: [])
-        return api.createAuthenticatedJSONRequest(url: endpoints.syncPatch, method: .PATCH, authToken: try getToken(), json: body)
+        let headers = ["Content-Encoding": "gzip"]
+
+        let body = try JSONSerialization.data(withJSONObject: json, options: []).gzipped()
```

Comment:
As discussed on zoom, we 1) could proceed here even if gzipping fails 2) it would be good to know if it fails - it is not actionable, but information that certain volume of patches fail could be a signal itself.